src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -227,21 +213,37 @@ function InteractiveMediaEditor({ content, onContentChanged, publicStorage, priv
               <RadioButton key={ratio} value={ratio}>{ratio}</RadioButton>
             ))}
           </RadioGroup>
-        </Form.Item>
-        <Form.Item label={t('common:videoDisplay')} {...formItemLayout}>
+        </FormItem>
+        <FormItem label={t('common:videoDisplay')} {...formItemLayout}>
           <Switch
             size="small"
             checked={showVideo}
             onChange={handleShowVideoChanged}
             disabled={![MEDIA_TYPE.video, MEDIA_TYPE.none].includes(getMediaType(sourceUrl))}
             />
-        </Form.Item>
-        <Form.Item label={t('common:width')} {...formItemLayout}>
+        </FormItem>
+        <FormItem label={t('playbackRange')} {...formItemLayout}>
+          <div className="u-input-and-button">
+            <Input
+              value={t('playbackRangeInfo', {
+                from: sourceStartTimecode ? formatMillisecondsAsDuration(sourceStartTimecode) : 'start',
+                to: sourceStopTimecode ? formatMillisecondsAsDuration(sourceStopTimecode) : 'end'
+              })}
+              readOnly
+              />
+            <MediaRangeSelector
+              sourceUrl={sourceUrl}
```

Comment:
Thanks, very good catch! Corrected.

Reply:
Since we pretty much do the same thing on line 44, we could have a `composeFullSourceUrl` method to use in both 🤔

Reply:
Good catch again, fixed. Maybe one day we will create a util function instead of having this or similar repeated in 50% of the plugins - or - get to the point where we always save full URLs in the plugin data model ;-)

Reply:
Yes, we need a global util for it for sure. Saving in the data model I don't think would be good though, I find it best to compose it dynamically based on current settings honestly.